docs/cli/install.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Install .NET Aspire CLI
+description: Learn how to install .NET Aspire CLI, which is a .NET global tool. Use the .NET Aspire CLI to create, run, and manage .NET Aspire projects.
+author: adegeo
+ms.author: adegeo
+ms.topic: install-set-up-deploy
+ms.date: 06/26/2025
+
+#customer intent: As a developer, I want to install the .NET Aspire CLI so that I can create, run, and manage .NET Aspire projects.
+
+---
+
+# Install .NET Aspire CLI
+
+This article teaches you how to install .NET Aspire CLI, which is a .NET global tool used to manage your .NET Aspire projects.
+
+## Prerequisites
+
+- [.NET SDK 8.0](https://dotnet.microsoft.com/download/dotnet/8.0) or [.NET SDK 9.0](https://dotnet.microsoft.com/download/dotnet/9.0).
```

Comment:
CLI will require .NET 9.0 SDK.

Reply:
... we are merging a change which will result in an error if the SDK is not installed when you use the CLi without the SDK installed (even if it is the native CLI).